internal/pkg/dynamic-config/dynamic_context.go

```diff
@@ -183,7 +183,7 @@ func (d *DynamicContext) SchemaRegistryCluster(cmd *cobra.Command) (*v1.SchemaRe
 			if len(srClusters) != 0 {
 				cluster = makeSRCluster(&srClusters[0])
 			} else {
-				cluster = nil
+				return nil, errors.NewSRNotEnabledError()
```

Comment:
if returning right here, it'll pass updating the config starting line 193, when the cluster is missing details but not nil. can you provide more testing / assessment / reasoning that we can just return here?

Reply:
Yeah, looks like we should just store the error instead and allow the updating the config to happen.